appveyor.yml

```diff
@@ -71,3 +74,10 @@ deploy:
     artifact: Pretzel.Logic.nupkg, Pretzel.Logic.symbols.nupkg
     on:
       appveyor_repo_tag: true
+
+  - provider: NuGet
+    api_key:
+      secure: NmM24h0TPsfINvy/1Qr4iJkmRjVztuY5umCJ6mzpxSjvuKl48AXjGEcgow+iJwL8
+    artifact: Pretzel.Tool.nupkg, Pretzel.Tool.nupkg
```

Comment:
No need to upload it twice :)

Reply:
Sorry I was really tired yesterday. It was supposed to include the symbols package.